modules/integration/tests-integration/tests-es/src/test/java/org/wso2/carbon/registry/es/notes/GregCustomRxtNotesTestESTestCase.java

```diff
@@ -0,0 +1,221 @@
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.carbon.registry.es.notes;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.wink.client.ClientResponse;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.carbon.registry.core.exceptions.RegistryException;
+import org.wso2.carbon.registry.es.utils.ESTestCommonUtils;
+import org.wso2.carbon.registry.es.utils.GregESTestBaseTest;
+import org.wso2.carbon.registry.resource.stub.ResourceAdminServiceExceptionException;
+import org.wso2.greg.integration.common.clients.ResourceAdminServiceClient;
+import org.wso2.greg.integration.common.utils.GenericRestClient;
+
+import javax.activation.DataHandler;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GregCustomRxtNotesTestESTestCase extends GregESTestBaseTest {
+    private static final Log log = LogFactory.getLog(GregCustomRxtNotesTestESTestCase.class);
+    private TestUserMode userMode;
+    String jSessionId;
+    String assetId;
+    String assetType;
+    String cookieHeaderPublisher;
+    GenericRestClient genericRestClient;
+    Map<String, String> queryParamMap;
+    Map<String, String> headerMap;
+    String publisherUrl;
+    String resourcePath;
+    String noteName;
+    String noteOverviewHash;
+    String noteAssetId;
+    String replyAssetId;
+    ESTestCommonUtils esTestCommonUtils;
+    private ResourceAdminServiceClient resourceAdminServiceClient;
+
+    @Factory(dataProvider = "userModeProvider")
+    public GregCustomRxtNotesTestESTestCase(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init(userMode);
+        String session = getSessionCookie();
+        noteName = "testNote33";
+        assetType = "wsdl";
+        genericRestClient = new GenericRestClient();
+        queryParamMap = new HashMap<String, String>();
+        headerMap = new HashMap<String, String>();
+        resourcePath = FrameworkPathUtil.getSystemResourceLocation()
```

Comment:
There are formatting issues in this PR. Kindly requesting to correct those formatting issues.